server/src/main/java/nl/inl/blacklab/server/requesthandlers/SearchParameters.java

```diff
@@ -233,6 +237,21 @@ public int getInteger(String name) {
         }
     }
 
+    public int getNumberOfDocs() {
+        String filter = getString("filter");
+        if (StringUtils.isBlank(filter)) {
+            return 0;
+        }
+        //Experiment with this regex as well "(docId:\\p{Graph}+)+"
+        Pattern patt = Pattern.compile("([a-f0-9-]+[)|\\s])+");
```

Comment:
Could you past an example or two of what this is supposed to match as a comment?

Reply:
the docs ids in the request created by ann http://github.com/lexionai/Ann/blob/c170d75a2802677298e1c7c434dc19152cce2c0f/ann/backend/blacklab/client.py#L345-L345

Reply:
so something like this ```section:("Term") AND docId:(17e58226-50f9-42a4-a44d-6c85520b2b58 OR 02bd6d2a-7f69-4617-a8ec-bf197e77d748 OR 46e9f18d-06b8-477f-a102-808ff5cc1a7e OR 06f9b97b-08ed-4abc-a68e-71c2a1da41ad OR fd29e22f-5006-4bb5-8a57-6547e2cb35b3 OR 30bb7f2c-1986-45e7-a3df-763d40862561 OR b2d3a90c-fc4f-40f5-b93a-75a6b6a37152 OR f964725d-b45a-46c2-b565-9a98340d00da OR fd4fd7da-ca8f-41bf-a402-67f0a8f72afb OR fca18b82-2d37-4a67-9ca4-445f85de8545 OR e5bf04dc-93c9-4e1a-8058-abcf33cbb038 OR 6e70cf9b-970a-41e2-b84f-41fa03dd7d16 OR 22115081-4197-47a2-b35d-0cf33fecedad OR 3517e995-c5a2-4b8c-8a18-efcf501fc9e5 OR af7de390-dddd-47a1-ae91-0402726d281f OR 01ecaf12-fd16-4556-8d72-32bdfd64a826 OR 2e8064bc-432a-4277-8419-ce51e9c19f15 OR 0cedef5f-8a80-4782-8025-8e95b955e0cc OR 082ffc3d-c22c-4c92-a3b1-1e7473432e4d OR e81d4d74-e746-4a44-9f58-891785ff0c09 OR 3618e5a8-db3c-45d9-82ea-0c7c2058f195 OR 89083f47-5fa2-45df-a8b4-b6fcfa532b50 OR a62b920a-bb59-4861-ae08-a96621ca71b9 OR 61c2048a-02a4-4a4d-9b40-b75fd467e0f5 OR 0f9e49a4-492d-43cb-bc69-47553c8c3e9c)
```